src/main/java/io/rcktapp/api/handler/elastic/ElasticDb.java

```diff
@@ -169,6 +178,7 @@ private void buildAliasTables(String elasticName, JSObject jsIndex, Map<String,
                {
                   table = new Table(this, aliasName);
                   tableMap.put(aliasName, table);
+                  log.info("Created elastic map {} -> {}: ", aliasName, elasticName);
```

Comment:
We never knew what it was doing with the elastic bits before.  This shows us on startup.